include/odp/api/spec/crypto_types.h

```diff
@@ -134,6 +134,16 @@ typedef enum {
 	 */
 	ODP_CIPHER_ALG_SNOW3G_UEA2,
 
+	/** Confidentiality NEA4 algorithm
```

Comment:
I think the word order is weird here. "NEA4 confidentiality algorithm" would sound better. I know some old API definitions have the same weird word order, but it does not need to be the same here.